packages/ngx-errors/package/src/lib/errors-base.directive.ts

```diff
@@ -42,15 +42,16 @@ export abstract class NgxErrorsBase {
     return controlState;
   });
 
-  private registerResolvedControl = effect(
-    () => {
+  private registerResolvedControl = (() => {
```

Comment:
please remove the self-executing function. No need for it now that we don't do angular version checking